chia/consensus/multiprocess_validation.py

```diff
@@ -219,15 +219,15 @@ async def pre_validate_blocks_multiprocessing(
     # they won't actually be added to the underlying blockchain object
     blockchain = AugmentedBlockchain(block_records)
 
-    diff_ssis: List[Tuple[uint64, uint64]] = []
+    diff_ssis: List[ValidationState] = []
```

Comment:
Would it make sense to rename this now?

Reply:
yeah, probably. I'll do that

Reply:
actually, I take that back. I have another patch that removes this altogether anyway